markdown/developers/training_guide.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Training material guide
+subtitle: A guide to writing instructions for training material.
+---
+
+# Contributing a training guide to nf-core
+
+Thank you for helping us write a guide to using an aspect of nf-core.
+
+## Links
+
+- [Questions](#questions)
+- [Objectives](#objectives)
+- [Glossary](#glossary)
+- [Before I begin](#before-i-begin)
+- [How to](#how-to)
+  - [Bytesize](#bytesize-talk)
+  - [Walkthrough](#walkthrough)
+- [Key points](#key-points)
+- [References](#references)
+
+## Questions
+
+- How can I contribute a training guide to nf-core?
+- Where do I put the guide?
+- How do I structure content?
+
+## Objectives
+
+- Learn how to structure an nf-core guide.
+- Know where to submit the pull request.
+
+## Glossary
+
+<details>
+    <summary>Click here to expand the glossary of potentially unfamiliar terms</summary>
+    - **Learning Objective:** A predictive statement describing the specific outcomes that a training session is intended to achieve.
+    - **Key point:** An important take home message.
+    - **Cognitive load:** This relates to the amount of information that working memory can hold at one time.
+    - **Fork a repository:** This makes a copy of the repository to your personal GitHub workspace.
+    - **Make a branch:** A branch is another version of the files in your repository, including a history of how files were changed from the start of the branch to the current state of files.
+</details>
+
+## Before I begin
+
+A checklist of what you need to begin.
```

Comment:
Might be good to mention that everything is markdown, so you need to know that. For the hackathon we are recommending that people go to https://www.markdowntutorial.com/ if they're new to it.